docs-new/docs/graphql/core/security-disclosure/index.rst.txt

```diff
@@ -1,82 +0,0 @@
-<div class="meta" description="Hasura security vulnerability protocol"
-keywords="hasura, docs, security, security disclosure, vulnerability">
-
-</div>
-
-# Security Vulnerability Protocol
-
-<div class="contents" backlinks="none" depth="1" local="">
-
-Table of contents
-
-</div>
-
-## Introduction
```

Comment:
A comment is missed from the original rst file

`inspired and adapted from https://kubernetes.io/docs/reference/issues-security/security/ (https://github.com/kubernetes/website/blob/master/content/en/docs/reference/issues-security/security.md)`

Please add the comment too.

Reply:
Done!